src/daemon/database/schema.rs

```diff
@@ -57,6 +57,7 @@ CREATE TABLE vaults (
     received_at INTEGER NOT NULL,
     updated_at INTEGER NOT NULL,
     spend_txid BLOB,
+    cancel_txid BLOB,
```

Comment:
This was specific to the `Spend` for which you can't know the txid in advance. For the Cancel it's already stored in the `presigned_transactions` table (better), or you can re-generate it deterministically (fallback)